spec/lib/smoke_tests/gateway/user_details_spec.rb

```diff
@@ -0,0 +1,57 @@
+describe SmokeTests::Gateway::UserDetails do
+  let(:user_details) { DB[:userdetails] }
+
+  before { user_details.delete }
+
+  context "Deleting smoke test users" do
+    context "Given no test users" do
+      before do
+        user_details.insert(username: "foo", contact: "foo@example.com", created_at: Date.today - 1)
+        user_details.insert(username: "bar", contact: "bar@digital.cabinet-office.gov.uk", created_at: Date.today - 1)
+      end
+
+      it "does not delete any users" do
+        expect { subject.delete_users }.not_to(change { user_details.count })
+      end
+    end
+
+    context "Given one test user" do
+      before do
+        user_details.insert(username: "foo", contact: "foo@example.com", created_at: Date.today - 1)
+        user_details.insert(username: "bar", contact: "bar@digital.cabinet-office.gov.uk", created_at: Date.today - 1)
+        user_details.insert(username: "baz", contact: "govwifi-tests+baz@digital.cabinet-office.gov.uk", created_at: Date.today - 1)
+      end
+
+      it "does deletes only the old user record" do
```

Comment:
I think it should be this instead?
```suggestion
      it "does deletes only the test user record" do
```